AppStoreHW/AppStoreHW/BasketViewController.swift

```diff
@@ -1,13 +1,13 @@
 //
-//  ViewController.swift
+//  BasketViewController.swift
 //  AppStoreHW
 //
 //  Created by Александр Андреевич Щепелин on 06.10.2022.
 //
 
 import UIKit
-/// Стартовый экран приложения
-final class ViewController: UIViewController {
+/// экран корзины
+final class BasketViewController: UIViewController {
     
     // MARK: - IBOutlets
```

Comment:
```suggestion
    // MARK: - IBOutlets
```
удаляем марки которые не используем